R/rules-line_break.R

```diff
@@ -42,8 +42,51 @@ remove_line_break_before_round_closing <- function(pd) {
   pd
 }
 
+#' @importFrom rlang seq2
 add_line_break_after_pipe <- function(pd) {
   is_special <- pd$token %in% c("SPECIAL-PIPE")
   pd$lag_newlines[lag(is_special)] <- 1L
   pd
 }
+
+
+#' Set line break for multi-line function calls
+#' @param pd A parse table.
+#' @param except A character vector with tokens before "'('" that do not
+#' @name set_line_break_if_call_is_multi_line
+#' @importFrom rlang seq2
+NULL
+
+#' @describeIn set_line_break_if_call_is_multi_line Sets line break after
+#'   opening parenthesis.
+set_line_break_after_opening_if_call_is_multi_line <-
+  function(pd,
+           except_token = NULL) {
+  if (!is_function_call(pd)) return(pd)
+  npd <- nrow(pd)
+  if (npd == 3) {
```

Comment:
Because `seq2(3, npd - 1)` would throw an error otherwise. `npd = 3` is the case  of empty function calls like`call()`. Just making sure there this does not happen:
```r
call(
)
```
We could circumvent that using `seq2(3, npd)` instead, can't actually remember why I did not do that.